draft-ietf-quic-transport.md

```diff
@@ -805,46 +805,50 @@ describe the various QUIC frame types that can be present in a Regular packet.
-   +-------------------------+---------------------------------+
-   |  Data length (0 or 16)  |  Stream Data (per data length)  |
-   +-------------------------+---------------------------------+
-~~~
+* The leftmost bit must be set to 1, indicating that this is a STREAM frame.
 
-The STREAM frame header fields are as follows:
+* `F` is the FIN bit, which is used for stream termination.
 
-* Frame Type: The Frame Type byte is an 8-bit value containing various flags,
-  and is formatted as the following 8 bits: 1FDOOOSS.
+* The `D` bit indicates whether a Data Length field is present in the STREAM
+  header.  When set to 0, this field indicates that the Stream Data field
+  extends to the end of the packet.  When set to 1, this field indicates that
+  Data Length field contains the length (in bytes) of the Stream Data field.
+  The option to omit the length should only be used when the packet is a
+  "full-sized" packet, to avoid the risk of corruption via padding.
 
-  * The leftmost bit must be set to 1 indicating that this is a STREAM frame.
+* The `OOO` bits encode the length of the Offset header field as 0, 16, 24,
+  32, 40, 48, 56, or 64 bits long.
 
-  * 'F' is the FIN bit, which is used for stream termination.
+* The `SS` bits encode the length of the Stream ID header field as 8, 16, 24,
+  or 32 bits.  (DISCUSS: Consider making this 8, 16, 32, 64.)
 
-  * The 'D' bit indicates whether a Data Length field is present in the STREAM
-    header.  When set to 0, this field indicates that the Stream Data field
-    extends to the end of the packet.  When set to 1, this field indicates that
-    Data Length field contains the length (in bytes) of the Stream Data field.
-    The option to omit the length should only be used when the packet is a
-    "full-sized" packet, to avoid the risk of corruption via padding.
+A STREAM frame is shown below. 
 
-  * The 'OOO' bits encode the length of the Offset header field as 0, 16, 24,
-    32, 40, 48, 56, or 64 bits long.
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                    Stream ID (8/16/24/32)                   ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                Offset (0/16/24/32/40/48/56/64)              ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                      [Data Length (16)]                       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                        Stream Data (*)                      ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #stream-format title="STREAM Frame Format"}
 
-  * The 'SS' bits encode the length of the Stream ID header field as 8, 16, 24,
-    or 32 bits.  (DISCUSS: Consider making this 8, 16, 32, 64.)
+The STREAM frame payload contains the following fields:
```

Comment:
I don't think it makes sense to call this the payload. Why not just say "In addition to the Type byte, the STREAM frame contains the following fields"?